tests/test_utilities.py

```diff
@@ -576,8 +563,9 @@ def test_send_email_notification_message_with_site_defined_values(
         """
         enrolled_in['start'] = datetime.datetime.strptime(enrolled_in['start'], '%Y-%m-%d')
         enterprise_customer = EnterpriseCustomerFactory(name=enterprise_customer_name)
+        # since migration alread sets up fallback templates, use per customer template
         EnrollmentNotificationEmailTemplateFactory(
-            enterprise_customer=None,
+            enterprise_customer=enterprise_customer,
```

Comment:
can we set one of these tests up (or add a test) that doesn't just rely on the default?
Like I see for the fallbacks test we have logic using the Admin (well at least creating an Admin template too):
```
EnrollmentNotificationEmailTemplateFactory(            enterprise_customer=None,            template_type=SELF_ENROLL_EMAIL_TEMPLATE_TYPE        )        

EnrollmentNotificationEmailTemplateFactory(            enterprise_customer=None,            template_type=ADMIN_ENROLL_EMAIL_TEMPLATE_TYPE        )
```
But I don't really see us checking for Admin templates and since that one requires actually passing in a value I'm more curious about it than the default. Maybe that's in a different commit than the range I'm looking at this exact moment, so if we do have that already, feel free to ignore me!

Reply:
Test added for per customer admin template!